homeworks/sargis_vardanyan/HelloWorldMaven/pom.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>groupId</groupId>
+    <artifactId>JavaIntermediateTech42</artifactId>
+    <version>1.0-SNAPSHOT</version>
+
+    <dependencies>
+<!--        <dependency>-->
+<!--            <groupId>junit</groupId>-->
+<!--            <artifactId>junit</artifactId>-->
+<!--            <version>4.13.2</version>-->
+<!--            <scope>test</scope>-->
+<!--        </dependency>-->
+        <dependency>
+            <groupId>org.junit.jupiter</groupId>
+            <artifactId>junit-jupiter-api</artifactId>
```

Comment:
only having this dependency isn't enough, if you check maven test will not run unit tests. You need to include junit-jupiter artificat id for that

Reply:
I got that question, about why mvn test command show the this output(0 tests.....), but working with Intellij IDEA's tools everything works as it should be, and also what about <build></build>, and <plugins></plugins>, do I need them for this task, and if yes why?

Reply:
I'm not sure if I get you right, but Intellij runs the tests itself, it doesn't need maven plugin to run it, but the maven needs it

Reply:
first of all, intellij doesn't use maven to build the maven project (although it still parse pom.xml to find the dependencies, repos etc..). The plugin for maven is the part that responsible for doing real actions like compliation, jar creation, packaging, deploying etc (https://maven.apache.org/guides/introduction/introduction-to-plugins.html#:~:text=What%20is%20a%20Plugin%3F,documentation%2C%20and%20on%20and%20on.)
So maven plugin is responsible for running unit tests, so if you skip the plugin, the unit tests can be running from the intellij but not from the maven

Reply:
Thanks a lot